doc/swagger-api-description/api.yaml

```diff
@@ -173,69 +176,214 @@ paths:
           schema:
             $ref: '#/definitions/ErrorModel'
 
-definitions:
+  /users/{userUuid}/clouds:
+    post:
+      description: Allows one to add an external cloud account to an existing ariADDna's user. The User MUST be already registered at the cloud service to be added.
+      operationId: addExternalCloudAccount
+      parameters:
+        - in: path
+          name: userUuid
+          description: Providing user's UUID one can access his CloudSet and add there one more external cloud account.
+          type: string
+          format: uuid
+          required: true
+        - in: body
+          name: cloud
+          description: A required information about an external cloud service that a user wants to include to his ariADDna.
+          schema:
+           $ref: '#/definitions/Cloud'
+      responses:
+        '201':
+          description: New cloud account was added successfully.
+          schema:
+            $ref: '#/definitions/Cloud'
+        '409':
+          description: This cloud account already exists.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+        default:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorModel'
 
+    get:
+      description: Returns a list of clouds which a certain user has connected to his ariADDna account.
+      operationId: getExternalCloudsList
+      parameters:
+        - in: path
+          name: userUuid
+          description: Providing user's UUID one can access his CloudSet and add retrieve the list of available external cloud accounts.
+          type: string
+          format: uuid
+          required: true
+      responses:
+        '200':
+          description: The list was found.
+          schema:
+            $ref: '#/definitions/CloudSetPages'
+
+  /users/{userUuid}/clouds/{cloudUuid}:
```

Comment:
We wont have simple API. Please move path "/users/{userUuid}/clouds/{cloudUuid}" to "/clouds/{cloudUuid}" without any other change.